src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -98,6 +98,11 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
     [assetId, isSymAssetWithdraw],
   )
 
+  const isNativeThorchainAsset = useMemo(
+    () => fromAssetId(assetId).chainId === thorchainChainId,
```

Comment:
might just add `|| isSymAssetWithdraw` and remove from the conditions below and update to `isNativeThorchainTx` to match vernacular